packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Changelog
 
 ## Unreleased
+* Changed
+  * Removed ability to set variable row heights
 * Added
   * Added masked cell support.
-
-* Changed
   * Removed ability to set variable row heights.
   * Updated styling on cell focus.
```

Comment:
It looks like the first line got duplicated. Also, should these both be under the `Changed` section now that it's been moved?